### ISSUE TITLE
Long lines never reflow in test declarations.

### ISSUE THREAD

**Issue by dreid** [prettier/prettier#2910]
This seems to be related to https://github.com/prettier/prettier/pull/2877

**Prettier 1.7.1**
[Playground link](https://prettier.io/playground/#%7B%22content%22%3A%22describe(%5C%22foo%5C%22%2C%20()%20%3D%3E%20%7B%5Cn%20%20const%20MINIMAL_STATE%20%3D%20%7B%20active_features%3A%20%5B%5D%2C%20csrf%3A%20%7B%20csrf_token_value%3A%20%5C%22csrf-token-value%5C%22%20%7D%2C%20successful_campaigns%3A%20%5B%5D%2C%20user%3A%20null%20%7D%3B%5Cn%7D)%3B%5Cn%5Cnit(%5C%22bar%5C%22%2C%20()%20%3D%3E%20%7B%5Cn%20%20%20%20const%20MINIMAL_STATE%20%3D%20%7B%20active_features%3A%20%5B%5D%2C%20csrf%3A%20%7B%20csrf_token_value%3A%20%5C%22csrf-token-value%5C%22%20%7D%2C%20successful_campaigns%3A%20%5B%5D%2C%20user%3A%20null%20%7D%3B%5Cn%7D)%3B%22%2C%22options%22%3A%7B%22ast%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%2C%22jsxBracketSameLine%22%3Afalse%2C%22output2%22%3Afalse%2C%22parser%22%3A%22flow%22%2C%22printWidth%22%3A80%2C%22semi%22%3Atrue%2C%22singleQuote%22%3Afalse%2C%22tabWidth%22%3A2%2C%22trailingComma%22%3A%22none%22%2C%22useTabs%22%3Afalse%7D%7D)
```sh
--parser flow
```

**Input:**
```jsx
describe("foo", () => {
  const MINIMAL_STATE = { active_features: [], csrf: { csrf_token_value: "csrf-token-value" }, successful_campaigns: [], user: null };
});

it("bar", () => {
    const MINIMAL_STATE = { active_features: [], csrf: { csrf_token_value: "csrf-token-value" }, successful_campaigns: [], user: null };
});
```

**Output:**
```jsx
describe("foo", () => {
  const MINIMAL_STATE = { active_features: [], csrf: { csrf_token_value: "csrf-token-value" }, successful_campaigns: [], user: null };
});

it("bar", () => {
  const MINIMAL_STATE = { active_features: [], csrf: { csrf_token_value: "csrf-token-value" }, successful_campaigns: [], user: null };
});

```

**Expected behavior:**

Long object literals and function calls should be broken up appropriately.

```jsx
describe("foo", () => {
  const MINIMAL_STATE = {
    active_features: [],
    csrf: { csrf_token_value: "csrf-token-value" },
    successful_campaigns: [],
    user: null
  };
});
```

**Comment by dreid** [comment]
This may be a duplicate of #2909

**Comment by lydell** [comment]
@duailibe

**Comment by duailibe** [comment]
Yikes, we should probably revert that 😞 was not the right fix, it seems

**Comment by suchipi** [comment]
Just released 1.7.2 where this is fixed. Sorry about that!

**Comment by suchipi** [comment]
Sorry that this stayed out there and broken for a few hours, everyone. I would have done a release earlier if I had been checking my GitHub notifications, but I have a day job 😅
